common-auth/src/main/java/com/hortonworks/registries/auth/server/KerberosAuthenticationHandler.java

```diff
@@ -461,8 +461,8 @@ protected static String getDoasUser(HttpServletRequest request) {
                         if (doAsUser.isEmpty()) {
                             return null;
                         }
+                        break;
```

Comment:
@guruchai : Will there be multiple "doAs" key in the header ?

Reply:
@guruchai So multiple "doAs" query parameter ?

Reply:
Sorry, with regards to the question, it does not make sense to have multiple doAs query parameters. Since the doAs is typically attached by a proxy server, it makes sense to pick the last one.
So break statement does not make sense. Thanks for pointing!